crates/telemetry-subscribers/src/bin/import_trace.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Do you want to rename this file to `import-trace.rs` so that the binary name is `import-trace` vs `import_trace` without needing configuration in the crate's Cargo.toml file?